src/external/token/interfaces/IERC20Issuance_blacklist_v1.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Imports
+
+// Internal
+import {IERC20Issuance_v1} from "@ex/token/IERC20Issuance_v1.sol";
+
+/**
+ * @title   ERC20 Issuance Token with Blacklist Interface
```

Comment:
This whole NatSpec section of an interface should be a 1-to-1 copy of the one from its contract - so we do not add the word "Interface" here.

Reply:
fixed